fuel-merkle/src/binary/root_calculator.rs

```diff
@@ -47,9 +49,10 @@ impl MerkleRootCalculator {
             return empty_sum().to_owned()
         }
         while self.stack.len() > 1 {
-            let right_child = self.stack.pop().expect("Unable to pop element from stack");
-            let left_child = self.stack.pop().expect("Unable to pop element from stack");
-            let merged_node = Node::create_node(&left_child, &right_child);
+            let right_child = self.stack.pop().expect("Checked in loop bound");
+            let left_child = self.stack.pop().expect("Checked in loop bound");
+            let merged_node = Node::create_node(&left_child, &right_child)
+                .expect("Unable to create a node");
```

Comment:
Can we explain why we expect this to not panic? (Same with above)

Reply:
Refactored away.